src/test/java/com/redhat/cloud/notifications/db/EmailAggregationResourcesTest.java

```diff
@@ -0,0 +1,112 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.TestLifecycleManager;
+import com.redhat.cloud.notifications.models.EmailAggregation;
+import com.redhat.cloud.notifications.models.EmailAggregationKey;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.vertx.core.json.JsonObject;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusTest
+@QuarkusTestResource(TestLifecycleManager.class)
+public class EmailAggregationResourcesTest extends DbIsolatedTest {
+
+    private static final ZoneId UTC = ZoneId.of("UTC");
```

Comment:
nitpick, we could use [ZoneOffset.UTC](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneOffset.html#UTC) which is more typesafe